ios/RCTVideo.m

```diff
@@ -485,8 +485,8 @@ - (void)playerItemDidReachEnd:(NSNotification *)notification
   }
 
   if (_repeat) {
-    AVPlayerItem *item = [notification object];
-    [item seekToTime:kCMTimeZero];
+    [self setSeek:0.0];
```

Comment:
I think that you need to do a comparison of `notification.object` and `_player.currentItem` here, or call `[notificationCenter removeObserver:self name:AVPlayerItemDidPlayToEndTimeNotification object:_player.currentItem]` at here https://github.com/kentos/react-native-video/blob/62ba9f18da218e87fb3d10ec7a6f421eb39f35f2/ios/RCTVideo.m#L285 in order not to process notifications of other videos